packages/react-ui-core/package.json

```diff
@@ -40,7 +40,7 @@
     "babel-plugin-transform-runtime": "6.23.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "6.24.1",
-    "date-fns": "^1.29.0",
+    "date-fns": "2.0.0-alpha.16",
```

Comment:
This is a newer version than the one we're using in AG.js. Do we really need 2 different versions?

Reply:
It will get upgraded once this is merged and published.  This is a dev dep here. It's listing `^2.0.0` as a peer dep.

Reply:
You mean AG will get upgraded, I assume.